torch_frame/data/multi_embedding_tensor.py

```diff
@@ -196,3 +206,32 @@ def cat(
             # which is inconsistent with when dim=0
             offset = torch.LongTensor(offset_list)
             return MultiEmbeddingTensor(num_rows, num_cols, values, offset)
+
+    def _normalize_index(
+        self,
+        index: Union[int, Tensor],
+        dim: int,
+        is_slice_end: bool = False,
+    ) -> Union[int, Tensor]:
+        """Helper function to map negative indices to positive indices and
+        raise :obj:`IndexError` when necessary.
+
+        Args:
+            index: Union[int, Tensor]: Input :obj:`index` with potentially
+                negative elements.
+            is_slice_end (bool): Whether a given index (int) is slice or not.
```

Comment:
Though not very sure, should it be `is slice end or not`?
```suggestion
            is_slice_end (bool): Whether a given index (int) is slice end or not.
```

Reply:
Done!

Reply:
Thanks!